tools/cdn.sh

```diff
@@ -1,27 +1,45 @@
-CURRENT_VERSION=$(node tools/attribute.js version)
+VERSION="version"
+MAJORMINOR="majorminor"
+
+CURRENT_VERSION=$(node tools/attribute.js $VERSION)
 EXTENSION_NAME="auth0-account-link"
+PACKAGE_JSON='../package.json'
+MAJORMINOR_VERSION=$(node tools/attribute.js $VERSION $PACKAGE_JSON $MAJORMINOR)
```

Comment:
Ends up making this a more `version` specific tool, if this isn't being used elsewhere maybe worth leaning into that? The `majorminor` parameter to template the response on seems a bit like an awkward add-on right now.

Reply:
@gausnes are you suggesting renaming attribute.js to version.js? and then getting rid of the `version` parameter so it always fetches the version attribute?

Reply:
> @gausnes are you suggesting renaming attribute.js to version.js? and then getting rid of the version parameter so it always fetches the version attribute?

Effectively, seems like the extensions of this script do not apply to other attributes with this extension.